omc3/tbt/handler.py

```diff
@@ -74,7 +74,7 @@ def _matrices_to_array(tbt_data):
 def _add_noise(data, noise):
     if noise <= 0.0:
```

Comment:
looking at the rest of the code, I think this should read `if noise is None`

Reply:
the check for None is outside of the method, we can remove the condition completely, thus allowing for negative numbers as well as @JoschD mentioned.